COMMUNICATON.md

```diff
@@ -0,0 +1,33 @@
+# Communication
+
+## Code of conduct
+
+This project has adopted the [GitHub community guidelines](https://help.github.com/en/github/site-policy/github-community-guidelines).
+
+## Issues
+
+Issues are used as the primary method for tracking anything to do with the Fairlearn project.
+
+See the [README](https://github.com/fairlearn/fairlearn/blob/master/CONTRIBUTING.md#issues)
+for information on how to file an issue if you think you have found a bug or have a feature request.
+
+## Proposals
+
+Design proposals for larger changes go into the [fairlearn-proposals repository](https://github.com/fairlearn/fairlearn-proposals). If you want to submit a proposal, just create a pull request there and maintainers will review it.
+
+## Microsoft Teams
```

Comment:
Really not a big deal, but I wish it was on gitter.

Reply:
I LOVE that you're bringing this up! @MiroDudik connected me with some other folks as well who felt similarly. I hope you see the irony if I tell you that I just started a survey of options in Teams itself :-D 
I am super interested in your input there (or in this PR if that's simpler for you)

In general, I'd like to figure such major communications changes out as soon as possible. Now it would disrupt a few people, whereas waiting would mean that we disrupt everyone who joins by then additionally.

Reply:
I'll hold off on this part until we've settled this :-)

Reply:
... update: I'll hold off until https://github.com/fairlearn/fairlearn-community/pull/7 is completed.

Reply:
We're now live on Gitter!